File: Entities/Concrete/Customer.cs
﻿using Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete
{
    public class Customer : ProjectBaseEntity, IEntity
    {
        public string Name { get; set; }
        public string Code { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public virtual List<Order> Orders { get; set; }

    }
}

File: UIPreparation/UI/src/app/core/components/app/customer/models/customer.ts
import { Order } from "../../order/models/Order";

export class Customer{
    id?:number; 
createdUserId?:number; 
createdDate?:(Date | any); 
lastUpdatedUserId?:number; 
lastUpdatedDate?:(Date | any); 
status:boolean; 
isDeleted:boolean; 
name?:string; 
code?:string; 
address?:string; 
phoneNumber?:string; 
email?:string; 
orders?:Order[]; 

}